src/components/transactions/Faucet/FaucetActions.tsx

```diff
@@ -13,13 +13,13 @@ export type FaucetActionsProps = {
 };
 
 export const FaucetActions = ({ poolReserve, isWrongNetwork, blocked }: FaucetActionsProps) => {
-  const { faucetService } = useTxBuilderContext();
+  const mint = useRootStore((state) => state.mint);
   const { currentAccount } = useWeb3Context();
```

Comment:
I noticed in a few places `useWeb3Context` is used to get the `currentAccount`, should this be refactored to use the root store?